Assignment/SampleData.cs

```diff
@@ -1,30 +1,75 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
+        public enum Column
+        {
+            Id, FirstName, LastName, Email, StreetAddress, City, State, Zip
+        }
+
         // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        public IEnumerable<string> CsvRows => File.ReadAllLines("C:\\Users\\Sam\\source\\repos\\EWU-CSCD371-2019-Fall2\\Assignment\\People.csv").Skip(1);

```

Comment:
```suggestion
        public IEnumerable<string> CsvRows => File.ReadAllLines("People.csv").Skip(1);
```
This should help fix your test cases from failing - the previous code will only work when it is run from Sam's machine.